canonicalwebteam/discourse_docs/parsers.py

```diff
@@ -50,11 +65,17 @@ def parse_index(topic):
 
     # Get the nav
     index["body_html"] = str(
-        get_preamble(index_soup, break_on_title="Content")
+        get_preamble(index_soup, break_on_title="Navigation")
     )
 
+    # Parse URL mapping
+    index["url_map"] = parse_url_map(index_soup)
+    # Add the homepage path
```

Comment:
I guess this is the homepage only for that topic?

Reply:
Sorry? There is one index topic, and the content of that topic can be accessed at `/`.

Reply:
Ok, got it now :)